config/config.php

```diff
@@ -91,6 +91,9 @@
 /**
  * Notification Center Gateways
  */
+if (!isset($GLOBALS['NOTIFICATION_CENTER']['GATEWAY'])) {
+    $GLOBALS['NOTIFICATION_CENTER']['GATEWAY'] = [];
+}
 $GLOBALS['NOTIFICATION_CENTER']['GATEWAY'] = array_merge(
     (array) $GLOBALS['NOTIFICATION_CENTER']['GATEWAY'],
```

Comment:
To be consistent with Contao, shouldn't we fix it like that?

```suggestion
    (array) ($GLOBALS['NOTIFICATION_CENTER']['GATEWAY'] ?? null),
```

Reply:
Yes, but in composer.json there is still PHP 5.4 as allowed requirement!

Reply:
🤦 😂